.dockerignore

```diff
@@ -9,6 +9,15 @@
 cache.sqlite
 README.md
 HACKING.md
+renovate.json
+LICENCE
+.sass-lint.yml
```

Comment:
I think we need the sass-lint file otherwise sass linting won't work with `./run`? Please test

Reply:
Nope. We are not using sass-lint anywhere in the build (package.json) therefore, multistage builds isn't using it either.

In any case, I have QA'ed with this and builds successfully.